insights/specs/default.py

```diff
@@ -436,6 +436,7 @@ def md5chk_file_list(broker):
         return ["/etc/pki/product/69.pem", "/etc/pki/product-default/69.pem", "/usr/lib/libsoftokn3.so", "/usr/lib64/libsoftokn3.so", "/usr/lib/libfreeblpriv3.so", "/usr/lib64/libfreeblpriv3.so"]
     md5chk_files = foreach_execute(md5chk_file_list, "/usr/bin/md5sum %s")
     mdstat = simple_file("/proc/mdstat")
+    mdadm_E = foreach_execute(mdstat, "/usr/sbin/mdadm -E %s")
```

Comment:
The output of the `cat /proc/mdstat` looks like:
```
Personalities : [raid1] [raid6] [raid5] [raid4]
md1 : active raid1 sdb2[1] sda2[0]
      136448 blocks [2/2] [UU]

md2 : active raid1 sdb3[1] sda3[0]
      129596288 blocks [2/2] [UU]

md3 : active raid5 sdl1[9] sdk1[8] sdj1[7] sdi1[6] sdh1[5] sdg1[4] sdf1[3] sde1[2] sdd1[1] sdc1[0]
      1318680576 blocks level 5, 1024k chunk, algorithm 2 [10/10] [UUUUUUUUUU]

md0 : active raid1 sdb1[1] sda1[0]
      16787776 blocks [2/2] [UU]

unused devices: <none> 
```
I don't think the `foreach_execute` would work for this `mdadm_E` spec.

You need to leverage the result of the [MdStat](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/mdstat.py#L60) parser to implement a suitable datasource to return the required list for the `mdadm` command, but not use the `mdstat` spec directly.